apps/evm/lib/evm/functions.ex

```diff
@@ -175,4 +181,24 @@ defmodule EVM.Functions do
   end
 
   defp is_invalid_jump_destination?(_operation, _inputs, _machine_code), do: false
+
+  defp static_state_modification?(:call, [_, _, value, _, _, _, _]), do: value > 0
```

Comment:
Do you think it makes sense for these `static_state_modification?` functions to live in their respective operation modules? I'm not sure that it _does_ make sense since I'm not quite sure which module should have knowledge about what a `static_state_modification` is, but I'm commenting here just in case that makes sense to you.

Reply:
It makes sense to put some kind of flag to operation group metadata module but I don't know how to store them flag based on input params for `call` operation

Reply:
Could we define it as a function that takes the input params? ie

    %EVM.Operation.Metadata{
       id: 0xFE,
       description: "Designated invalid instruction. It can be used to abort the execution.",
       sym: :invalid,
       input_count: 0,
       output_count: 0,
       group: :system,
       is_static?: fn (_, _, value, _, _, _, _) ->
        value > 0
       end
    },

Not sure how much metaprogramming and complexity it'll take to make this work. Might be worth a shot ¯\_(ツ)_/¯

Reply:
it looks like anonymous functions are not supported as struct fields
```
** (ArgumentError) invalid value for struct field is_static, cannot escape #Function<0.102477801 in file:lib/evm/ope
ration/metadata.ex>. The supported values are: lists, tuples, maps, atoms, numbers, bitstrings, PIDs and remote func
tions in the format &Mod.fun/arity      
```